packages/frontend/src/contexts/game/reducer/index.js

```diff
@@ -1,5 +1,6 @@
 import { handleMatch } from 'helpers/handleMatch';
 import { resetTiles } from 'helpers/resetTiles';
+import { awardPoint } from 'helpers/awardPoint';
```

Comment:
These can all be imported into `helpers/index.js` and then exported from there to allow them to all be imported from the same place.
```javascript
import { awardPoint, handleMatch, resetTiles } from 'helpers';
```